tests/extmod/tls_dtls.exp

```diff
@@ -0,0 +1 @@
+OK
```

Comment:
This file needs to be called `tls_dtls.py.exp`.  Also please add a newline on the last line (but not a trailing blank line!).